### PR TITLE
Create entrypoint and bin for api

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -23,6 +23,9 @@
   },
   "typings": "./lib/index.d.ts",
   "license": "MIT",
+  "bin": {
```
Comment (Itrulia) [inline packages/api/package.json:26]: Could also just be:

`"bin": "./bin/run-server.js"``, then it can be called via `npx @wepublish/api`

**Comment by tomaszdurka** [comment]
@Itrulia please re-review

**Comment by Itrulia** [comment]
Looks good!

**Comment by Itrulia** [comment]
@tomaszdurka the merge broke the build

**Pull request merged**